src/cli.ts

```diff
@@ -89,13 +93,20 @@ const splitDirection = args["--split-vertically"]
   ? "horizontally"
   : undefined;
 
+let optionsFromFile: InitialOptions = {};
+try {
+  optionsFromFile = JSON.parse(fs.readFileSync("./.newsh.json").toString());
+} catch (e) {}
+
 const initialOptions: InitialOptions = {
   env: {},
   cwd: undefined,
-  cd: args["--cd"],
-  split: !!splitDirection,
-  splitDirection,
-  terminalApp: args["--terminalApp"]
+  cd: args["--cd"] || optionsFromFile.cd,
+  split: !!(splitDirection || optionsFromFile.splitDirection),
+  splitDirection: splitDirection || optionsFromFile.splitDirection,
+  terminalApp: args["--terminalApp"] || optionsFromFile.terminalApp,
+  terminalAppSetup:
+    args["--terminalAppSetup"] || optionsFromFile.terminalAppSetup
```

Comment:
The logic for merging the config options with the cli options is always the same, CLI wins.

As we would like to use the same logic for the node-api, i'll suggest to implement it in `normalize.ts` file.